platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/Address.java

```diff
@@ -148,7 +148,7 @@ private byte[] clone(byte[] x) {
      * @param id                  the ID for that member
      * @param nickname            the name given to that member by the member creating this address
      * @param selfName            the name given to that member by themself
-     * @param stake               the amount of stake (0 if they should have no influence on the consensus)
+     * @param weight               the amount of weight (0 if they should have no influence on the consensus)
```

Comment:
spacing off.
`* @param weight               the amount of weight (0 if they should have no influence on the consensus)`
should be
`* @param weight              the amount of weight (0 if they should have no influence on the consensus)`

Reply:
removing the space.